goverge/coverage.py

```diff
@@ -205,7 +205,8 @@ def get_package_deps(project_package, test_path, tag):
     package_deps = [
         package.replace("]", "").replace("[", "").replace("'", "")
         for package in list(set(output))
-        if project_package.split("\n")[0] in package]
+        if project_package.split("\n")[0] in package and
+        project_package.split("\n")[0]+"/vendor" not in package]
```

Comment:
You are right I could make this cleaner. 

I would say in some cases performance is an issue a single package can have a very large number of dependencies because it searches recursively.

Not sure how much of a difference it would make but I will find a cleaner way of doing this because the current is pretty messy and unclear what is going on.